ngraph_onnx/onnx_importer/backend.py

```diff
@@ -39,6 +39,13 @@
 class NgraphBackend(Backend):
     """Takes an ONNX model with inputs, perform a computation, and then return the output."""
 
+    _supported_devices = [
```

Comment:
This list should be populated dynamically, as it will depend on how `libngraph.so` was compiled and what hardware is actually available.

Reply:
You may need to map nGraph device names to ONNX device names.